src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3882,10 +3880,6 @@ private bool SelfDescribingEvents
         // WARNING: Do not depend upon initialized statics during creation of EventSources, as it is possible for creation of an EventSource to trigger
         // creation of yet another EventSource.  When this happens, these statics may not yet be initialized.
         // Rather than depending on initialized statics, use lazy initialization to ensure that the statics are initialized exactly when they are needed.
-
-        // used for generating GUID from eventsource name
```

Comment:
Does it make sense to retain this comment on the local variable above?

Reply:
IMO I'd leave the PR how it is now, but I am fine either way.